src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,96 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+public class StringCalculatorTest {
+
+  private StringCalculator stringCalculator;
+
+  @BeforeEach
+  void init() {
+    stringCalculator = new StringCalculator();
+  }
+
+  @Test
+  @DisplayName("덧셈 테스트")
+  void addTest() {
+    assertThat(stringCalculator.calculate("1 + 1")).isEqualTo(2);
+  }
+
+  @Test
+  @DisplayName("연산 횟수 3개 이상 테스트")
+  void addTripleTest() {
+    assertThat(stringCalculator.calculate("1 + 1 + 1 + 1")).isEqualTo(4);
+  }
+
+  @Test
+  @DisplayName("뻴셈 테스트")
+  void minasTest() {
+    assertThat(stringCalculator.calculate("2 - 1")).isEqualTo(1);
+  }
+
+  @Test
+  @DisplayName("곱셈 테스트")
+  void multipleTest() {
+    assertThat(stringCalculator.calculate("2 * 4")).isEqualTo(8);
+  }
+
+  @Test
+  @DisplayName("나눗셈 테스트")
+  void divTest() {
+    assertThat(stringCalculator.calculate("4 / 2")).isEqualTo(2);
+  }
+
+  @Test
+  @DisplayName("복합 테스트")
+  void complexTest() {
+    assertThat(stringCalculator.calculate("4 / 2 * 3")).isEqualTo(6);
+  }
+
+  @ParameterizedTest
+  @DisplayName("입력 값이 null이거나 빈 공백 문자일 경우")
+  @NullAndEmptySource
+  void blankCheckTest(String input) {
+    assertThatExceptionOfType(RuntimeException.class)
+        .isThrownBy(() -> {
+          stringCalculator.calculate(input);
+        }).withMessageMatching("null 또는 공백을 입력하실 수 없습니다.");
```

Comment:
개인적으로는 람다식의 바디가 하나의 실행문을 갖는다면 하나의 라인으로 표현하는 것이 
가독성 측면에서 좋을 것 같다는 생각이 드는데 용훈님께서는 어떻게 생각하시나요?
```suggestion
        .isThrownBy(() -> stringCalculator.calculate(input))
        .withMessageMatching("null 또는 공백을 입력하실 수 없습니다.");
```